app/pages/clinicworkspace/ClinicPatients.js

```diff
@@ -421,13 +418,37 @@ export const ClinicPatients = (props) => {
     </Box>
   );
 
-  const renderEdit = patient => (
-    <Icon className="edit-clinic-patient" icon={EditIcon} label={'Edit'} variant={'button'} onClick={handleEditPatient(patient)} />
-  );
+  const renderMore = patient => {
+    const items = [];
+
+    items.push({
+      icon: EditIcon,
+      iconLabel: t('Edit Patient Information'),
+      iconPosition: 'left',
+      id: `edit-${patient.id}`,
+      variant: 'actionListItem',
+      onClick: _popupState => {
+        _popupState.close();
+        handleEditPatient(patient);
+      },
+      text: t('Edit Patient Information'),
+    });
 
-  const renderRemove = patient => (
-    <Icon className="remove-clinic-patient" icon={DeleteIcon} label={'Remove'} variant={'button'} onClick={handleRemove(patient)} />
-  );
+    if (isClinicAdmin) items.push({
+      icon: DeleteIcon,
+      iconLabel: t('Remove Patient'),
+      iconPosition: 'left',
+      id: `delete-${patient.id}`,
+      variant: 'actionListItemDanger',
+      onClick: _popupState => {
+        _popupState.close();
+        handleRemove(patient);
+      },
+      text: t('Remove Patient')
+    });
+
+    return <PopoverMenu id="action-menu" items={items} />
```

Comment:
```suggestion
    return <PopoverMenu id={`action-menu-${patient.id}`} items={items} />
```
Should avoid duplicate `id` prop here.

Reply:
Good call 👍🏼